spec/lib/tasks/fixtures/find-eu-exit-guidance-business.json

```diff
@@ -0,0 +1,74 @@
+{
+	"base_path": "/find-eu-exit-guidance-business",
```

Comment:
The indentation here is using tabs, could they be replaced with two spaces to maintain consistency with other code

Reply:
Good spot - I just took the default output from jsonlint.com